#Message for a 2D bounding box, any for 4 of these arrtibutes are sufficient
#to describe it.

float64 x1
float64 y1
float64 x2
float64 y2
